src/main/java/com/flansmod/client/model/ModelVehicle.java

```diff
@@ -26,6 +26,17 @@
 	public ModelRendererTurbo rightTrackWheelModels[] = new ModelRendererTurbo[0];	//These go with the tracks but rotate
 	public ModelRendererTurbo leftTrackWheelModels[] = new ModelRendererTurbo[0];
 	
+	public ModelRendererTurbo leftAnimTrackModel[][] = new ModelRendererTurbo[0][0];  //Unlimited frame track animations
+	public ModelRendererTurbo rightAnimTrackModel[][] = new ModelRendererTurbo[0][0];
+	
+	public ModelRendererTurbo rightAnimTrackModel1[] = new ModelRendererTurbo[0]; //3 frame track animation
```

Comment:
I would rather this be done properly in code and that Toolbox figures out its own way to interface with it